testdata/push/push.go

```diff
@@ -0,0 +1,179 @@
+package main
+
+import (
+	"flag"
+	"fmt"
+	"io/fs"
+	"log"
+	"os"
+
+	"github.com/google/go-containerregistry/pkg/crane"
+	v1 "github.com/google/go-containerregistry/pkg/v1"
+	"github.com/google/go-containerregistry/pkg/v1/mutate"
+	"github.com/spf13/pflag"
+	"gopkg.in/yaml.v2"
+)
+
+const (
+	bundlesSubPath  string = "bundles"
+	catalogsSubPath string = "catalogs"
+)
+
+func main() {
+	var (
+		registryAddr string
+		imagesPath   string
+	)
+	flag.StringVar(&registryAddr, "registry-address", ":12345", "The address the registry binds to.")
+	flag.StringVar(&imagesPath, "images-path", "/images", "Image directory path")
+	pflag.CommandLine.AddGoFlagSet(flag.CommandLine)
+	pflag.Parse()
+
+	log.Printf("registry configured with path %s, listening on %s", imagesPath, registryAddr)
+
+	bundlesFullPath := fmt.Sprintf("%s/%s", imagesPath, bundlesSubPath)
+	catalogsFullPath := fmt.Sprintf("%s/%s", imagesPath, catalogsSubPath)
+
+	bundles, err := buildBundles(bundlesFullPath)
+	if err != nil {
+		log.Fatalf("failed to build bundles: %s", err.Error())
+	}
+	catalogs, err := buildCatalogs(catalogsFullPath)
+	if err != nil {
+		log.Fatalf("failed to build catalogs: %s", err.Error())
+	}
+
+	// Push the images
+	// TODO without insecure option
```

Comment:
Ugh? Make an issue?

Reply:
I've fixed this - `push` job is using the mounted certs now.